src/components/Monitoring/dashboard/PVBatteryChart.tsx

```diff
@@ -214,12 +214,12 @@ export default function PVBatteryChart(props: PVBatteryChartProps): JSX.Element
     const getChartData = async () => {
       if (selectedPVBatteryPeriod) {
         const timePeriodParams = getTimePeriodParams(selectedPVBatteryPeriod, timeZone);
-        const startTime = timePeriodParams.start;
-        const endTime = timePeriodParams.end;
-        if (BMU) {
+        const startTime = timePeriodParams.start.toISO();
+        const endTime = timePeriodParams.end.toISO();
+        if (BMU && startTime && endTime) {
```

Comment:
If it is possible with no errors, can we change this to

```
if (BMU && startTime.isValid && endTime.isValid) {
  const response = await getTimeSeriesHistory(
    startTime.toISO(),
    endTime.toISO(),
   ...
```

And remove toISO from lines 217 and 218.

Same for TemperatureHumidityChart (note that isValid is not a function just a property on DateTime)